extensions/vertx-http/deployment/src/test/resources/styles.css

```diff
@@ -0,0 +1,3 @@
+.quarkus-blue {
```

Comment:
Can we use a dummy text file instead?

Reply:
Hi @cescoffier, I will use the file that has been used by StaticHandler test, WDYT?